cars/v1/vanilla/templates/config/elasticsearch.yml

```diff
@@ -69,12 +69,27 @@ transport.port: {{transport_port}}
 # The default list of hosts is ["127.0.0.1", "[::1]"]
 #
 {%- if all_node_ips %}
+  {%- if all_node_ips_count is defined %}
+    {%- if all_node_ips_count > 1 %}
```

Comment:
I was hesitant to set a default value since `all_node_ips_count` should only be undefined if using older versions of Rally. Defaulting to `0` in this case will incorrectly leave `discovery.seed_hosts` unset when there are multiple master nodes. This was the reason behind the redundant declaration on line 86.

Another attempt is on the way.